### PR TITLE
Revive pvwatts norm

### PR DESCRIPTION
**Pull request by mdeceglie**
- [x] Code changes are covered by tests
- [x] ~New functions added to `__init__.py`~
- [x] API.rst is up to date, along with other sphinx docs pages
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog

### CODE REVIEW

**Comment by mdeceglie** [comment]
In response to @GregoryKimball 's [feedback](https://github.com/NREL/rdtools/pull/214#discussion_r505815129) on #214 and subsequent discussions, this PR proposes to roll back the deprecation of the PVWatts-based normalization functions.

The idea is to provide rudimentary modeling for users who may not wish to develop a separate model. We would keep the deprecations of the SAPM methods.

File: docs/sphinx/source/changelog/pending.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+*************************
+Pending
+*************************
+
+Deprecations
+------------
+* The the deprecation of :py:func:`~rdtools.normalization.pvwatts_dc_power`,
+  and :py:func:`~rdtools.normalization.normalize_with_pvwatts` has been reversed.
+  (:pull:'225')
```
Comment (kanderso-nrel) [inline docs/sphinx/source/changelog/pending.rst:9]: This is #227

Comment (kanderso-nrel) [reply docs/sphinx/source/changelog/pending.rst:9]: Also needs backticks on the `:pull:`, not singlequotes

**Review by kanderso-nrel — APPROVED** [review]
De-deprecation LGTM, minor note below

File: docs/sphinx/source/changelog/pending.rst (hunk 1 above)
Comment (mdeceglie) [reply docs/sphinx/source/changelog/pending.rst]: Good catches, thanks!

**Review by cdeline — APPROVED** [review]
Looks good, thank you Mike!

**Pull request merged**